pom.xml

```diff
@@ -543,44 +543,6 @@
             <artifactId>unirest-java</artifactId>
             <version>1.4.9</version>
         </dependency>
-        <!-- BagIt export -->
-        <dependency>
-			<groupId>org.apache.commons</groupId>
-			<artifactId>commons-compress</artifactId>
-			<version>1.18</version>
-		</dependency>
```

Comment:
Unless this is the cause (probably isn't), I'd leave it (the commons-compress) in. It's used in the /BagIt export classes. They aren't called outside the DuraCloud archiver right now, but not having this dependency will break those classes.